man/systemd.network.xml

```diff
@@ -1004,6 +1004,26 @@ DuplicateAddressDetection=none</programlisting></para>
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>MulticastIGMPVersion=</varname></term>
+        <listitem>
+          <para>Configures IPv4 Multicast IGMP Version to be used, and controls the value of
+          <filename>/proc/sys/net/ipv4/conf/<replaceable>INTERFACE</replaceable>/force_igmp_version</filename>.
+          If not configure not change the IPV4 Multicast IGMP protocol version. Takes one of <literal>no</literal>,
```

Comment:
Please drop the sentence `If not ...` here.

But add `Defaults to unset, and the sysctl value will be unchanged.` at the end of the paragraph.

Reply:
done